installer/dns/setup.sh

```diff
@@ -44,17 +47,19 @@ if [ -z "$result" ]; then
   fi
 
   header "Waiting in the loop for updated dns records"
-  result=""
-  wait_time=0
-  while [ -z "$result" ]
-  do         
-    result=`ping foo.$DOMAIN_NAME -c 1 2>&1|grep $EXTERNAL_IP`
-    sleep 20
-    wait_time=$(($wait_time + 20))
-    print_ne "\r\r\r\r\r\r\r\r\r\r\r\r elapsed time: $wait_time s"
-  done
+  cd ~/inference-model-manager/installer/utils/route53/
+  virtualenv .venvaws -p python3
+  . .venvaws/bin/activate
+  pip install awscli --upgrade 
+  export AWS_DNS=`./apply.sh CREATE $EXTERNAL_IP ${CLUSTER_NAME_SHORT}.nlpnp.adsdcsp.com`
```

Comment:
What if we're not installing platform via CI? There's no parameter for CLUSTER_NAME_SHORT